targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_MCU_K64F/drivers/fsl_dspi.c

```diff
@@ -576,13 +576,13 @@ void DSPI_MasterTransferCreateHandle(SPI_Type *base,
     handle->userData = userData;
 }
 
-status_t DSPI_MasterTransferBlocking(SPI_Type *base, dspi_transfer_t *transfer)
+status_t DSPI_MasterTransferBlocking(SPI_Type *base, dspi_transfer_t *transfer, char dummy)
 {
     assert(transfer);
 
     uint16_t wordToSend = 0;
     uint16_t wordReceived = 0;
-    uint8_t dummyData = DSPI_DUMMY_DATA;
```

Comment:
@sg- Addition to code was done by @geky to improve the SPI read/write performance. But I believe no one is using it still. In case of SD, issue is only during SPI read.
Options:
 1) Revert  SPI read to byte reads in SD driver. Read performance will be affected.
 2) Report this bug to NXP, and change the define  DSPI_DUMMY_DATA (Hoping they will change it in their SDK as well).
 3) Please suggest :-)